src/applications/personalization/profile/tests/components/direct-deposit/BankInfoForm.unit.spec.jsx

```diff
@@ -36,10 +33,6 @@ describe('<BankInfoForm/>', () => {
   it('should render a SchemaForm', () => {
     expect(schemaForm.exists()).to.be.true;
   });
-  it("should set the SchemaForm's schema  and uiSchema correctly", () => {
```

Comment:
This test was failing after my change and I am deleting it because I don't like it 😆 It's a prime example of testing implementation details. And it now fails because there is no longer a static `schema` or `uiSchema` defined alongside the `BankInfoForm`. But `BankInfoForm` still does exactly what it's supposed to do. A more useful test would be to make sure that the `BankInfoForm` renders the form inputs we expect it to and they have the correct IDs.